docs/userGuide/deployingTheSite.md

```diff
@@ -329,10 +329,162 @@ Here are the steps to set up Netlify:
 
 Now your site will be deployed on Netlify at the given address specified after deployment. It will be updated automatically when the default branch of your repo is updated.
 
-Additionally, **when contributors make a pull request to your GitHub repo, you can _preview_ the updated site** at the bottom of the pull request by clicking on `details` link in the PR:
+## Previewing Pull Requests for MarkBind sites
+
+There may be more than one user making changes to your MarkBind site. In such cases, it might be prudent to have a workflow that is centered around pull requests (PRs) such as a [feature branch workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) or a [forking workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow). 
+
+You can setup PR previews in order to automatically build and deploy the modified Markbind site based on the changes in the PR, which would lead to greater convenience and time-savings for the PR reviewers. 
+
+### Previewing PRs using Netlify
+
+By following the [steps to deploy to Netlify](#deploying-to-netlify) in the previous section, you would automatically be able to preview PRs.
+
+You can _preview_ the updated site at the bottom of the pull request by clicking on `details` link in the PR:
 
 <include src="screenshot.md" boilerplate var-alt="Preview deploy" var-file="netlifyPreview4.png" inline />
 
+For more information on previewing PRs with Netlify, you may refer to [Netlify's docs](https://www.netlify.com/tags/deploy-previews/).
+
+### Previewing PRs using Surge
+
+You may also preview PRs using [Surge](https://surge.sh/), which is an NPM package that does static web publishing. Here are the steps to do so:
+
+1. First install Surge using by typing `npm install --global surge` on your terminal. 
+1. Next, type `surge` in the terminal. You should see the following prompt:
+
+    <include src="screenshot.md" boilerplate var-alt="Create Surge account" var-file="surgeCreateAccount.png" inline />
+
+1. Proceed to create a Surge account.
```

Comment:
Perhaps can also include instruction to inform user to press 'Enter' to proceed to use the default value of project and domain when setting up the account? New users may be confused by this step.

![image](https://user-images.githubusercontent.com/47470981/109462762-f217ce80-7a9e-11eb-8b5d-6d464ffb64ed.png)

Reply:
Actually this part is not needed - the user can just `CTRL-C` out of this process. I have added a clarification in the userGuide on this. Could you help to check to see if it is clearer?

Reply:
> Actually this part is not needed - the user can just CTRL-C out of this process (after logging in / creating an account).

Ahh I see. Yup it's much clearer now 👍 Thanks for updating :)